backend/python/tapa/tapac.py

```diff
@@ -172,7 +172,8 @@ def create_parser() -> argparse.ArgumentParser:
       type=argparse.FileType('r'),
       dest='directive',
       metavar='file',
-      help=('Use a specific partitioning directive json file '
+      help=('[obsolete] Please use --floorplan-pre-assignments instead.\n'
```

Comment:
I think we should keep `--directive` as it is (in terms of functionality). `--directive` is different from `--floorplan-pre-assignments`. `--directive` must be a full floorplan, that is generated either by autobridge or manually. This is very helpful to provide reproducible results; as far as I know, autobridge results may not be reproducible (and may require a Gurobi license). The name is confusing, though. I think we should add an alias for `--directive`, like `--floorplan`, `--floorplan-override`, or `--full-floorplan`.

Reply:
With floorplan-pre-assignments we can reproduce the result ---- we pre-assign the location of every vertex.
we should not combine the vertex-to-region mapping with how the slots are connected. Tapa should deal with pure data flow programs, and all device-related part (e.g., process the "topology" of how slots are connected) will be pushed into autobridge later.